FreeRTOS/FreeRTOS-Plus-TCP
# Addition of MSP432E401Y EMAC driver

Third-party code for an MSP432E401Y NetworkInterface driver.

Description
-----------
This is an example driver for the Texas Instruments SimpleLink MSP432E401Y microcontroller with internal Ethernet transceiver.  Code to setup the driver is provided in the README.md file.  Additional middleware is also provided. 

Test Steps
-----------
Tested using UDP and TCP sockets, ping floods, nmap and in the context of a FreeRTOS+TCP application.

Related Issue
-----------
https://forums.freertos.org/t/freertos-tcp-crashing-on-cortex-m4f-and-possible-bug-msp432e401y/11606/17

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
